centaur/src/main/resources/standardTestCases/bt_343/bt_343.wdl

```diff
@@ -0,0 +1,86 @@
+version 1.0
```

Comment:
I'll remove this WDL from the PR and add it to the ticket for future reference as I suspect this PR will not be the last in this space.